### ISSUE TITLE
Paul Tol accessible palettes

### ISSUE THREAD

**Issue by bwiernik** [easystats/see#353]
Increasingly popular accessible palettes: https://cran.r-project.org/web/packages/khroma/vignettes/tol.html#introduction

**Comment by DominiqueMakowski** [comment]
I like all of them, aside from bright, muted, and pale & dark. Particular fan of the diverging ones! thanks for bringing that to my knowledge :)

**Comment by strengejacke** [comment]
What would you suggest, copying some of the schemes into see?

**Comment by bwiernik** [comment]
Yeah, I was thinking palette_tol() with the various options for the "palette" argument like some of our others. I can plan on taking care of this
